### PR TITLE
New Progress Component according to Bootstrap Italia

### LINKED ISSUE #155: [BUG] 'Progress' component has to be updated and renamed
**Describe the bug**
'Progress' component is out-of-date.

**To Reproduce**
1. Go to 'Progress' > 'Esempi"
2. Compare it to ['Bootstrap Italia Progress' component](https://italia.github.io/bootstrap-italia/docs/componenti/progress-indicators/)
3. See error

**Expected behavior**
It should behave accordingly to Bootstrap Italia component, and renamed to "Progress Indicators"

### PR DESCRIPTION
**Pull request by Diuda**
Fixes #155

#### PR Checklist

- [x] My branch is up-to-date with the Upstream `next` branch.
- [ ] The unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

This refactors the current **Progress** component according to [Bootstrap Italia Progress Component](https://italia.github.io/bootstrap-italia/docs/componenti/progress-indicators/)
The name of the component is also changed to Progress Indicator
Will be updating new spinners and circular bar in future PR

### CODE REVIEW

**Comment by sanyamdogra** [comment]
Hey @Diuda, have you pull `next` branch to latest as checks are failing. Try resolving these, Thanks.

**Comment by Diuda** [comment]
> Hey @Diuda, have you pull `next` branch to latest as checks are failing. Try resolving these, Thanks
 
Yeah, my branch is synced with `next` branch. No idea why the tests are failing. Looking into it

File: stories/Progress/Progress.stories.js [hunk 1]
```diff
@@ -19,48 +19,86 @@ import Altezza from "./Altezza.md";
...
 stories.addDecorator(withA11y);
 
 const EsempiComponent = () => (
     <div>
-        <Progress value="0" className="mt-3" style={{ height: 10 }}/>
-        <Progress value="25" className="mt-3" style={{ height: 10 }}/>
-        <Progress value={50} className="mt-3" style={{ height: 10 }}/>
-        <Progress value={75} className="mt-3" style={{ height: 10 }}/>
-        <Progress value="100" className="mt-3" style={{ height: 10 }}/>
+        <div className="progress" style={{marginBottom: "2em"}}>
```
Comment (sanyamdogra) [inline stories/Progress/Progress.stories.js]: I don't think we need to apply `style` property here.

File: stories/Progress/Progress.stories.js [hunk 2]
```diff
@@ -115,8 +115,7 @@ const EsempiInterattiviComponent = () => {
                   <div className="progress-bar-label">
                         <span className="sr-only">Progresso </span>{value}%
                   </div>
-
-                  <div className="progress progress-color" style={{marginBottom: "2em", height:`${height}px`}}>
+                  <div className="progress progress-color">
                         <div className={`progress-bar bg-${color}`} role="progressbar" style={{width: `${value}%`}} aria-valuenow={`${value}`} aria-valuemin="0" aria-valuemax="100">{label}</div>
```
Comment (sanyamdogra) [inline stories/Progress/Progress.stories.js]: Yup! looks good. As far as Label or Etichette is considered in the Interactive component, it is not displayed due to the height of progress bar. You may merge it and I will fix it and add other progress indicators.

**Pull request merged**